include/librealsense2/h/rs_types.h

```diff
@@ -39,6 +39,18 @@ typedef enum rs2_exception_type
 } rs2_exception_type;
 const char* rs2_exception_type_to_string(rs2_exception_type type);
 
+/** \brief Image Clockwise Rotation Degrees */
+typedef enum rs2_clockwise_rotation_degrees
```

Comment:
Is it used?

Reply:
Yes, it is used by resolve_requests(...), open(...) and probe_and_commit(...) methods.
We still need it because if we rotating the frames we should know it and ask from the device the appropriate resolutions and in other side we need to switch between width and height in the frame callback.